libs/components/lists/testing/src/modules/list-summary/list-summary-harness.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { provideNoopAnimations } from '@angular/platform-browser/animations';
+import { SkyListSummaryModule } from '@skyux/lists';
+
+import { SkyListSummaryHarness } from './list-summary-harness';
+import { SkyListSummaryItemHarness } from './list-summary-item-harness';
+
+@Component({
+  imports: [SkyListSummaryModule],
+  template: `
+    <sky-list-summary data-sky-id="test-summary">
+      <sky-list-summary-item
+        data-sky-id="test-item-1"
+        [labelText]="'Total Items'"
+        [value]="42"
+      />
+      <sky-list-summary-item
+        data-sky-id="test-item-2"
+        [labelText]="'Average Score'"
```

Comment:
```suggestion
        [labelText]="'Average score'"
```
While going back through, I noticed that I missed this headline-style capitalization,